frontend/src/components/Lobby/Lobby.js

```diff
@@ -0,0 +1,36 @@
+import "./Lobby.scss";
+import React from "react";
+import { useHistory } from "react-router-dom";
+import GoBackButtonLobby from "../GoBackButton/GoBackButton";
+
+const Lobby = () => {
+  const history = useHistory();
+
+  const ws = new WebSocket("ws:localhost/api/lobby");
```

Comment:
This should be in `useEffect` with an empty dependency array that also handles cleaning up this web socket. As it stands, this code will create a new web socket connection a lot of times.